showcase/src/app/demo/header/header.component.html

```diff
@@ -0,0 +1,26 @@
+<mat-card>
+	<h1>Stark App Header Guidelines</h1>
+	<p>Stark provide you usefull classes to implement your application header. You just need to include the header theme in your SCSS</p>
+	<stark-pretty-print [data]="importSCSS"
+			format="css"
+			[enableHighlighting]="true"></stark-pretty-print>
+	<p>Then you can copy/paste this HTML into your code</p>
+		<stark-pretty-print [data]="headerHtml"
+			format="html"
+			[enableHighlighting]="true"></stark-pretty-print>
+			<p>The <b>stark-app-header</b> provide a background to the header</p>
+			<p>
+				The <b>stark-app-bar</b> contains three containers <b>stark-content-left</b>, <b>stark-content-center</b>, <b>stark-content-right</b> which split the header in three parts.
+				All the direct children of those 3 containers will be aligned in a column.
+			</p>
+			<p>
+				<b>stark-content-left</b> contains 2 containers <b>stark-actions</b> for the navigation actions and <b>stark-page-title</b> which displays the current page title.
+			</p>
+			<p>
+				<b>stark-content-center</b> contains the app logo
+			</p>
+			<p>
+					<b>stark-content-right</b> contains 3 containers, the first one for the App Data component and the logout button, the second for language selection and <b>stark-actions-alt</b> which contains some alternative actions.
```

Comment:
Maybe you can put these 3 paragraphs in a list so it is displayed with bullets instead ;)